src/ORM/Behavior.php

```diff
@@ -325,7 +325,7 @@ public function implementedEvents(): array
     public function implementedFinders(): array
     {
         $methods = $this->getConfig('implementedFinders');
-        if (isset($methods)) {
+        if ($methods !== null) {
```

Comment:
```suggestion
        if (!$methods) {
```

It could be set to an empty array.

Reply:
but the isset() would have not caught that either. So we are OK with the behavior change here?

Reply:
it returns the empty array afterwards anyway, right? so we probably can keep the null check

Reply:
Actually, it should be 

    if ($methods) {

I fixed that up.

Reply:
I am still not sure what the empty array being excluded now means :)